core/files/css/base.php

```diff
@@ -666,6 +666,32 @@ protected function parse_content() {
 		return $this->get_stylesheet()->__toString();
 	}
 
+
+	private function update_background_images_dimension( $control, $values ) {
+		$background_dimensions_prefix = 'background_dimensions';
+		if ( false !== strpos( $control['name'], 'background_image' ) ) {
```

Comment:
All of my comments from this method's JS counterpart apply here too

Reply:
Not relevant anymore, the logic has changed.